### DISCUSSION TITLE
Change the status to a button for users to export library by clicking it.

### DISCUSSION THREAD

**Discussion by earthmoon** [retorquere/zotero-better-bibtex#2645] [Ideas]
Hello,

When we set BetterBibTeX to automatically export library when idle, there are status bar on the upper-right conor of the Zotero:
![image](https://github.com/retorquere/zotero-better-bibtex/assets/45897471/889ac6f3-45b1-471e-975d-91dccb1b5b3d)

May you add a button here to let users to click to export the library as sometimes we can't wait idle time to export.

Thanks.

**Comment by retorquere** [comment]
There is an option in the preferences to do immediate export for an auto-export, and there is also a menu option to do it if you right-click the library or collection for which you set up an auto-export.

**Reply by earthmoon** [reply]
I just wondering what is you refering of ** there is also a menu option to do it if you right-click the library or collection for which you set up an auto-export**.

I right-click the collection which I have set for automatically export, then what should I choose?
![image](https://github.com/retorquere/zotero-better-bibtex/assets/45897471/ce28e8cf-6e86-47e7-8433-cb1b3febc55c)

**Reply by retorquere** [reply]
It's only visible if autoexport is set to idle or off.

**Reply by retorquere** [reply]
Did find a bug in it, looking at that now.

**Reply by retorquere** [reply]
Release build 118 is underway

**Reply by earthmoon** [reply]
Okay.
